index.html

```diff
@@ -1,12 +1,113 @@
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+    <meta name="description" content="Резюме"/>
+    <meta name="keywords" content="Резюме"/>
+    <!--	<link href="style.css" type="text/css" rel="stylesheet"/>-->
+    <style>
+        body {
+            background-image: url(background.png);
+        }
+
+        div {
+            width: 60%;
+            margin: 0 auto;
+            text-align: center;
+            padding: 5px 0 0 0;
+            background: url(http://oboi.cc/1600-900-100-uploads/11_05_2013/view/201209/oboik.ru_21869.jpg) repeat;
+        }
+
+        h1 {
+            text-align: center;
+        }
+
+        img {
+            width: 30%;
+            height: 30%;
+        }
+
+        .one ul {
+            list-style: none;
+            margin: 0 auto;
+        }
+
+        a {
+            text-decoration: none;
+        }
+
+        nav {
+            display: block;
+            width: 387px;
+            margin: 0 auto 20px;
+        }
 
-<h1>This is a Heading</h1>
-<p>This is a paragraph.</p>
+        .one ul {
+            padding: 1em 0;
+            background: #ECDAD6;
+        }
 
+        .one a {
+            padding: 1em;
+            background: rgba(177, 152, 145, .3);
+            border-right: 1px solid #b19891;
+            color: #695753;
+        }
+
+        .one li {
+            display: inline;
+        }
+
+        ol, ul {
+            list-style-position: inside
+        }
+
+        p {
+            padding: 5px;
+            margin: 5px;
+            font-size: 17px;
+        }
+
+        h3 {
+            font-size: 20px;
+            padding: 5px;
+            margin: 5px;
+        }
+
+    </style>
+</head>
+<body>
+<h1>Моя розповідь про себе</h1>
+<div>
+    <img src="i.jpg" alt="Моє фото">
+    <p class="text">Мене звати Сергій<br>Мені 18 років</p>
+    <h3>Місце навчання:</h3>
+    <p><a href="http://www.cdu.edu.ua" target="_blank"> ЧНУ імені Б. Хмельницького</a><br> 2 курс<br>Факультет <a
+            href="https://fotius.cdu.edu.ua" target="_blank"> ОТІУС</a></p>
+    <h3>Що я знаю у програмуванні:</h3>
+    <ol>
+        <li>Основи С++</li>
+        <li>Основи HTML</li>
+        <li>Основи CSS</li>
+        <li>Вивчаю С#</li>
+        <li>Вивчаю ООП</li>
+    </ol>
+    <h3>Чим займаюсь у вільний час:</h3>
+    <ul>
+        <li>Граю на гітарі/електрогітарі</li>
+        <li>Вивчаю різні мови програмування</li>
+        <li>Подорожую</li>
+    </ul>
+    <h3>Я у соціальних мережах:)</h3>
+    <nav class="one">
+        <ul>
+            <li><a href="http://vk.com/sergeyyatsyn" target="_blank">Вконтакте</a></li>
+            <li><a href="https://www.instagram.com/sergeyyatsyn" target="_blank">Instagram</a></li>
+            <li><a href="https://t.me/sergeyyatsyn" target="_blank">Telegram</a></li>
+            <li><a href="https://github.com/SerhiiYatsyn" target="_blank">GitHub</a></li>
+        </ul>
+    </nav>
+</div>
 </body>
-</html>
+</html>
```

Comment:
live empty line in the end of the file. Setup your IDE to add it on save by default